src/utility/cloneMatrix.js

```diff
@@ -0,0 +1,8 @@
+export default function cloneMatrix(mSource) {
```

Comment:
what's mSource? could be named better, I think.

Reply:
I got told it is common to name it source in that case.